corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -76,11 +77,12 @@ def build_case_tile_detail(self):
 
         # Add case search action if needed
         if module_offers_search(self.module) and not module_uses_inline_search(self.module):
-            from corehq.apps.app_manager.suite_xml.sections.details import DetailContributor
             detail.actions.append(
                 DetailContributor.get_case_search_action(self.module, self.build_profile_id, self.detail_id)
             )
 
+        DetailContributor.add_no_items_text(self.app, detail, self.detail_type, self.module)
```

Comment:
nit:
Just thinking of readability `DetailContributor.add_no_items_text` would make me think the DetailContributor object is being changed but in actuality, the detail being passed in is. It might be clearer to just `get_no_items_text ` and add it to detail separately.

Reply:
I was actually thinking to also change the method like this
```
@staticmethod
def get_no_items_text(app, detail_type, module):
    if detail_type.endswith('short') and app.supports_empty_case_list_text:
        return Text(locale_id=id_strings.no_items_text_detail(module))      
```
but then the caller will have to do 
```
detail.no_items_text = get_no_items_text(app, detail_type, module)
```
But I do like the naming change you did